.circleci/config.yml

```diff
@@ -57,8 +60,46 @@ jobs:
       - store_artifacts:
           path: /tmp/test-results
           destination: tests
+  deploy:
+    docker: 
+      - image: circleci/python:3.7.1
+    steps:
+    - docker-cache/set-deploy-tag
+    - run:
+        command: |
+          export AWS_ACCESS_KEY_ID=${DEPLOY_TAG}_AWS_ACCESS_KEY_ID
+          export AWS_SECRET_ACCESS_KEY=${DEPLOY_TAG}_AWS_SECRET_ACCESS_KEY
+    - aws-cli/install
+    - aws-cli/configure
+    - run:
+        name: Update ECS service
+        environment:
+          APP_NAME: meadow
+          ECS_CLUSTER: meadow
+          ECS_SERVICE: meadow
+          ECS_TASK: meadow-app
+          ECS_CONTAINER: meadow-app
+        command: |
+          networkconfig=$(aws ecs describe-services --cluster ${ECS_CLUSTER} --service ${ECS_SERVICE} | jq -cM '.services[0].networkConfiguration')
+          overrides="{\"containerOverrides\":[{\"name\":\"${ECS_CONTAINER}\",\"command\":[\"eval\",\"Meadow.ReleaseTasks.migrate()\"]}]}"
+          aws ecs run-task --cluster ${ECS_CLUSTER} --task-definition ${ECS_TASK} --overrides ${overrides} --launch-type FARGATE --network-configuration ${networkconfig}
+          aws ecs update-service --cluster ${ECS_CLUSTER} --service ${ECS_SERVICE} --force-new-deployment        
 workflows:
-  version: 2
   ci:
     jobs:
       - test
+      - docker-cache/publish:
+          requires:
+            - test
+          filters:
+            branches:
+              only:
+                - /^deploy\/.+/
```

Comment:
Will this eventually need to be `/^master|deploy\/.+/`?

Reply:
The branch list is an array, so it will eventually be:

```yaml
          filters:
            branches:
              only:
                - master
                - /^deploy\/.+/
```

Reply:
Cool, thanks!